### ISSUE TITLE
duplicate of introduction and examples in Redoc documentation

### ISSUE THREAD

**Issue by juleshaas** [actinia-org/actinia-core#306]
The introduction and examples appear twice in https://redocly.github.io/redoc/?url=https://actinia.mundialis.de/latest/swagger.json.

**Comment by mmacata** [comment]
I cannot reproduce it locally. I have all plugins installed:
```
{
    "api_version": "3.0.1",
    "grass_version": {...},
    "plugin_versions": {
        "actinia_metadata_plugin": "1.0.2",
        "actinia_module_plugin": "2.2.4",
        "actinia_satellite_plugin": "0.0.4",
        "actinia_stac_plugin": "0.0.1",
        "actinia_statistic_plugin": "0.0.4"
    },
    "plugins": "actinia_statistic_plugin,actinia_satellite_plugin,actinia_metadata_plugin,actinia_module_plugin,actinia_stac_plugin",
    "python_version": "3.8.5 (default, Jul 20 2020, 23:11:29) - [GCC 9.3.0]",
    "running_since": "n/a",
    "version": "3.1.1.post0.dev5+g89a5035.dirty"
}
```
and when I open `http://localhost:8088/api/v3/swagger.json`, it appears only once. Also with `https://petstore.swagger.io/` it is only rendered once. So I assume it is a bug/feature of `https://redocly.github.io/`...
